kANe/v5p2/10_Hindu_Culture_and_Civilization/36_Characteristics_of_Hindu_Culture.md

```diff
@@ -102,93 +103,58 @@ The Mahābhārata passages quoted in n. 18 p. 8 of H. of Dh. Vol. II say 'the th
+Hindu thought on the three goals of human life came to this ' do your duty, do not fall into temptations, perform duties for their own sake ( Gita II, 47, III, 19 ), do unto others as you would be done by (Gitā VI. 32, Anuśāsanaparva 113. 8-9, śānti 259. 20. = Cr. ed. 251. 19), earn wealth but without conflict with dharma and without injury to others and lead a chaste sexual life and enjoy aesthetic pleasures. These summarise the teachings underlying the conceptions of the three puruṣārthas. There was hardly any real pessimism[^2614] in the principal Dharmaśāstra works except perhaps here and there in the Mahābhārata. They consider life to be worth living, provided all actions are informed by the dictates of Dharma. Manu ( XII. 88-89 ) provides that Karma (conduct or actions ) prescribed by the Veda is of two kinds, viz. _pravṛtta_ and _nivṛtta_, the former leading to happiness in this world and to _svarga_, and the latter to niḥśreyasa (mokṣa) and requiring complete absence of all hankerings preceded by realization of _brahma_. The Anuśāsanaparva ( chap. 146. 76-80 ) divides Dharma into pravṛttilakṣana (characterized by persistent activity) and nivṛttilakṣaṇa (characterised by abstention from worldly activities and desires), which latter is to be followed for mokṣa and lays down certain practical and salutary rules viz. one should constantly make gifts according to one's ability, constantly offer sacrifices and perform rites to secure one's prosperity. One should collect wealth following what is right and such wealth obtained righteously should be divided into three parts; one should secure both dharma and artha with one-third of the wealth acquired, one-third should be spent on Kāma (i.e, chaste sexual life and enjoyment of other pleasures not in conflict with dharma ) and one-third he should increase. Manu (VII. 99 and 101) prescribes similar rules for the king.
 
-2615. groft sa mara alami yana 1
+[^2614]: The Bible was very pessimistic, both Christ and Saint Paul regard the world as evil or at least spoiled. Christ calls upon people not to resist evil ( vide Hobhouse 'Morals in evolution 'Vol. II, p. 152).
```

Comment:
'Morals in evolution 'Vol. II, p. 152  > 'Morals in evolution' Vol. II, p. 152 (space placement)

Reply:
corrected.